### PR TITLE
Ingredient management: Update UGDG

### LINKED ISSUE #150: Update UGDG

### PR DESCRIPTION
**Pull request by rebstan97**
Closes #150

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -267,7 +267,9 @@ Password is masked by default and should never be shown
 ====
 // end::accountmanagement[]
 
-=== Adding an ingredient: `add-ingredient` or `add-ing`
+// tag::ingredientmanagement[]
+=== Ingredients Management
+==== Adding an ingredient: `add-ingredient` or `add-ing`
```
Comment (AZhiKai) [inline docs/UserGuide.adoc]: Would it be better if the alias is something like `ai`?
```suggestion
==== Adding an ingredient: `add-ingredient` or `ai`
```

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -764,6 +738,21 @@ Examples:
 * `delete-role r/999`
 * `dr id/999`
 
+==== Removing out-of-stock dishes from daily menu `[coming in v2.0]`
+
+Removes out-of-stock dishes that require ingredients with low stock count from the daily menu. +
+Format: `remove-oos-dish`
```
Comment (AZhiKai) [inline docs/UserGuide.adoc]: Add alias too.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (rebstan97) [reply docs/UserGuide.adoc]: `ai` is already the alias for `add-item`

Comment (AZhiKai) [reply docs/UserGuide.adoc]: I see, okay then no choice if that's the case.

**Comment by rebstan97** [comment]
> @rebstan97 remember to update your branch. :]

right, thanks for the reminder!

**Comment by AZhiKai** [comment]
Do you have any v2.0 stuffs? Do add it to the end of UG too.

**Comment by rebstan97** [comment]
> Do you have any v2.0 stuffs? Do add it to the end of UG too.

Yes I have already added them. Namely 5.29.4 and 5.29.5

**Comment by AZhiKai** [comment]
> > Do you have any v2.0 stuffs? Do add it to the end of UG too.
> 
> Yes I have already added them. Namely 5.29.4 and 5.29.5

Okay great, remembee to indent the sections btw. 
And for your 2.0, do add the format etc as well.

**Comment by AZhiKai** [comment]
> > Okay great, remembee to indent the sections btw.
> > And for your 2.0, do add the format etc as well.
> 
> What do you mean by indenting the sections?

See https://cs2103-ay1819s1-f10-4-menus.netlify.com/userguide

image

**Comment by rebstan97** [comment]
Yup I have done it already 👍

**Pull request merged**